### ISSUE TITLE
Fade away after a certain period

### ISSUE THREAD

**Issue by sqs** [sourcegraph/sourcegraph-token-highlights#3]
After hovering a token, the only way to make line highlights go away is to hover another token. Instead I would like it if they went away after I moved my mouse away from the code view entirely.

This is hard/impossible to do given the current APIs available to extensions, of course.

**Comment by chrismwendt** [comment]
Out of curiosity, do you find the highlights obtrusive/distracting? I'm wondering why you want them to go away. Would you prefer that they only show when clicking on a token?

The highlights will at least be less obtrusive once per-token highlighting is implemented.

VS Code only has settings to completely turn off highlights https://stackoverflow.com/questions/39775406/how-to-turn-off-matching-highlighting There doesn't appear to be any command or keyboard shortcut to hide them.

**Comment by sqs** [comment]
Yeah, they are a bit distracting. Subtler colors in the dark theme would help too.
